### PR TITLE
Moving SIG to TAG

### PR DESCRIPTION
**Pull request by amye**
Also adding a settings.yml file that TAG leads + TOC liaisons can update

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -65,20 +66,23 @@ If you want to suggest a specific topic or action item, please do some basic res
...
-[brendandburns]: https://github.com/brendandburns
+[resouer]: https://github.com/resouer
+[cdavisafc]: https://github.com/cdavisafc
 [halcyondude]:   https://github.com/halcyondude
 [RichiH]:        https://github.com/RichiH
 
 ## Governance
 
-This SIG follows [the standard operating model provided by the TOC](https://github.com/cncf/toc/blob/master/sigs/cncf-sigs.md#operating-model).
+This TAG follows [the standard operating model provided by the TOC](https://github.com/cncf/toc/blob/master/sigs/cncf-tags.md#operating-model).
```
Comment (paulfantom) [inline README.md:85]: Link is broken.

**Comment by RichiH** [comment]
Thanks for the catch, @paulfantom. This rename will break a ton of things over the next few weeks. The only way on our end is https://github.com/cncf/tag-observability/pull/34 and sit it out.

**Comment by amye** [comment]
This depends on https://github.com/cncf/toc/pull/654, when that's merged (shortly, just waiting on one more TAG), this will resolve.

**Comment by amye** [comment]
This should! hopefully! be resolved now.

**Comment by paulfantom** [comment]
> This should! hopefully! be resolved now.

https://github.com/cncf/toc/blob/main/tags/cncf-sigs.md

Nope, still 404 :( Just to note, In cncf/toc repository there is [tags/cncf-sigs.md](https://github.com/cncf/toc/blob/main/tags/cncf-sigs.md) but no [tags/cncf-tags.md](https://github.com/cncf/toc/blob/master/sigs/cncf-tags.md). Document from this PR references the latter.

**Pull request merged**